conf/nginx.conf

```diff
@@ -55,6 +55,8 @@ http {
 
     server {
         listen 80;
+        listen [::]:80;
```

Comment:
That section is only used for HTTP-> HTTPS redirect, and IPv6 is even [disabled in docker by default](https://docs.docker.com/config/daemon/ipv6/): the easiest way to listen on IPv6 on the container is to [forward IPv6 port to it](https://serverfault.com/a/1054917). Why do you think it's necessary to listen to IPv6 inside the container? Do you have a use case for it?

Reply:
A part of my infrastructure is IPv6-only. This section is also responsible for responding to Letsencrypt challenge, so the challenge failed until I enabled IPv6.

Also, in my case, I used host networking instead of docker's port proxying to reduce overhead (I expected the container to serve large files over a 10G network). 

Actually, I did not think about this, but it's probable that this directive will give an error in the default docker environment which has IPv6 disabled. In this case I agree this change should not be done.